processors/ttX3.py

```diff
@@ -192,17 +192,17 @@ def jetSelection(jetDict):
                 jetMinPt=30.,
                 jetMaxEta=2.4,
                 workingpoint = BTagSelection.TIGHT,
-                storeKinematics=[],
+                storeKinematics=["pt", "eta", "phi"],
                 storeTruthKeys = truthKeys,
             )
         )
         
     systNames = jetDict.keys()
    
-    #at least 2 AK4 jets
+    # At least 2 AK4 jets? No! We need at least 4 jets!
```

Comment:
I would leave this cut at >= 2 AK4 jets, as this is the requirement needed for Gabriele's study, where the spectator tops both decay dileptonically, hence two (b-)jets, and the resonance X decays into 2 fat jets (potentially top-tagged)

@matt-komm Maybe Vichayanun should have his own processor for the top-Yukawa study? Or should we maintain one single processor where we adjust the cuts ourselves outside GitHub to fit the different analyses we study?

Reply:
I can do some more filtering at the postprocessor level. This common processor can have a cut of 2+ AK4 jets, and I can further impose a cut for 4+ AK4 jets.

Reply:
@vicha-w  i agree with denise that you should make your own processor since there will be less overlap in the logic once the analyses get more mature